packages/components/MoveHandle.vue

```diff
@@ -71,16 +113,26 @@ export default Vue.extend({
     },
   },
   mounted() {
-    if (this.$el.parentElement) {
-      this.handleElement = this.$el.parentElement
-      return
-    }
-    console.error(
-      'MoveHandle: No parent element used, using MoveHandle container.'
-    )
-    this.handleElement = this.$el as HTMLElement
+    const handleElement = this.$el as HTMLDivElement
+    handleElement.style.position = 'fixed'
+    handleElement.style.width = '100%'
+    handleElement.style['z-index'] = 1
+    handleElement.style.left = '0'
+    handleElement.style.top = `${calculateTop(
+      top,
+      this.$root.$el.clientHeight,
+      this.maxHeight
+    )}px`
+    this.resizeObserver = new ResizeObserver(this.updateMaxHeight)
+    this.resizeObserver.observe(handleElement)
```

Comment:
I feel like we ought to unobserve the handleElement beforeDestroy, albeit I can't quite put my finger on how it could lead to erroneous behaviour not to do it. 🎩 at your discretion.

Reply:
Done 3e6fdce8